### ISSUE TITLE
The react PDF viewer removes all semantic elements so is not accessible.

### ISSUE THREAD

**Issue by lschiff** [eScholarship/jschol#968]
The react PDF viewer is not accessible--the way it renders the words on the page removes all semantic elements.  Andrew suggests that one alternative is to use an embed with the browser's default PDF viewer which is more accessible.

This is item 78 from the [accessibility report](https://docs.google.com/spreadsheets/d/1kIMiW09tBtk5d3kJjzUMSCo1tEWPn7bUc0BYAJes9Hw/edit?gid=0#gid=0)

**Comment by lschiff** [comment]
4/3/2026: Can't swap out the pdf viewer because there is no available in line PDF viewer that meets accessibility needs. However, the downloadable PDF files from the item view are machine readable because they are either text based or have OCR, and some are inherently accessible.  So our strategies will be to 1) identify viable remediation strategies; 2) continue exploring PDF viewer enhancements with an eye towards replacing the viewer on eScholarship; 3) explore generating more accessible alternative/derivative formats (e.g. HTML); and 4) evaluate the feasibility of having a "Request accessible version" button on all item pages.
